app/views/admin/schedules/index.html.haml

```diff
@@ -25,8 +25,24 @@
                 method: :get, class: 'btn btn-primary'
                 = link_to 'Delete', admin_conference_schedule_path(@conference.short_title, schedule.id),
                 method: :delete, class: 'btn btn-danger', data: { confirm: "Do you really want to delete Schedule #{schedule.id}?" }
-
-.row
-  .col-md-12.text-right
-    = link_to 'Add Schedule', admin_conference_schedules_path(@conference.short_title),
-    method: :post, class: 'btn btn-primary'
+- if @venue && @venue.rooms.any?
```

Comment:
Do you think we should prefer writing this like that?
Because it is less explicit and the same amount of code.